Sources/CardPayments/APIRequests/ConfirmPaymentSourceRequest.swift

```diff
@@ -36,12 +40,15 @@ struct ConfirmPaymentSourceRequest: APIRequest {
         path = String(format: pathFormat, orderID)
         
         jsonEncoder.keyEncodingStrategy = .convertToSnakeCase
-        body = try jsonEncoder.encode(confirmPaymentSource)
+        do {
+            body = try jsonEncoder.encode(confirmPaymentSource)
+        } catch {
+            throw CardClientError.encodingError
+        }
         
         // TODO - The complexity in this `init` signals to reconsider our use/design of the `APIRequest` protocol.
         // Existing pattern doesn't provide clear, testable interface for encoding JSON POST bodies.
     }
-    
```

Comment:
I think we want to keep this line break

Reply:
addressed in this commit 6409a89f4d28cb87a1025b5f579f2e4e4a2e1210